Dockerfile

```diff
@@ -82,4 +82,6 @@ RUN yarn install --production=false
 
 COPY . .
 RUN chown -R jenkins:jenkins /application
+RUN chmod -R 777 /application
```

Comment:
Can you give a little info on this change? I know the jenkins user workaround for volume permissions was wonky, but maybe there's a better way?